### PR TITLE
Update to k8s 1.18

### PR DESCRIPTION
**Pull request by aiyengar2**
Related Issue: https://github.com/rancher/rancher/issues/25117

### CODE REVIEW

File: pkg/controllers/management/kontainerdrivermetadata/data.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ const (
 
 const (
 	APIVersion           = "management.cattle.io/v3"
-	RancherVersionDev    = "2.4"
+	RancherVersionDev    = "2.4.3-rc100"
```
Comment (aiyengar2) [inline pkg/controllers/management/kontainerdrivermetadata/data.go:35]: Required as the minimum rancher version for k8s 1.18 is set to 2.4.3-rc99 in KDM:

https://github.com/aiyengar2/kontainer-driver-metadata/blob/afe5b43a12112a86eccee8281f89fd4d2e50ae4e/rke/k8s_version_info.go#L121

Comment (prachidamle) [reply pkg/controllers/management/kontainerdrivermetadata/data.go:35]: Then the value should be 2.4.3-rc99 I think

Comment (mrajashree) [reply pkg/controllers/management/kontainerdrivermetadata/data.go:35]: From offline discussion 2.4.3-rc100 was still showing 1.18 version, but changing ti to 2.4.3-rc99 seems correct

Comment (aiyengar2) [reply pkg/controllers/management/kontainerdrivermetadata/data.go]: Updated in the latest commit!

**Pull request merged**